start.py

```diff
@@ -0,0 +1,62 @@
+import GoogleApi
+import drive
+import sys
+import gmail
+
+
+# ******************************************************
+# *********** PUT CONFIGURATION FILE ID HERE ***********
+fileID = "18Hzz__cX7K3fv_tpZmJsa620EXYwPj0d"
+fileName = sys.argv[1]
```

Comment:
Ovo će ti pukne ako pozovem program bez argumenata, što će uvek biti slučaj.

Reply:
✔️
Radi bez unosa argumenata

Reply:
Jako bre radi, zar ne dobijas _index out of range_?

Reply:
Sad radi, commitovao sam na pogresan repo